auth4genai/ai/docs/get-started/overview.mdx

```diff
@@ -18,7 +18,7 @@ Leverage Auth0's Universal Login to enable AI agents to securely authenticate us
 Manage tokens to enable AI agents to call first-party APIs.
 
 <br />
-`NEXT.JS`  `FASTAPI` `VERCEL AI`
+`LANGGRAPH.JS` `VERCEL AI` `CLOUDFLARE AGENTS` `FASTAPI`
```

Comment:
Let's just use AI frameworks, else this is gonna be a long list as we add more quickstarts
```suggestion
`LANGGRAPH` `VERCEL AI` `CLOUDFLARE AGENTS`
```